.github/workflows/ci.yml

```diff
@@ -45,12 +49,15 @@ jobs:
           activate-environment: helios-dev
           environment-file: environment-dev.yml
           python-version: ${{ matrix.python }}
+          miniconda-version: "latest"
 
       - name: Install Helios
         run: |
           python -m pip install -v .
       
       - name: Run tests
+        if: runner.os != 'macOS' # Skip tests on macOS for now 
+        #because of a bug in laz files
```

Comment:
Please copy my entire comment from above - it is a bit more detailed about what goes wrong (we are writing corrupted LAZ).